lms/models/application_instance.py

```diff
@@ -18,6 +19,16 @@
 class ApplicationInstance(BASE):
     """Class to represent a single lms install."""
 
+    class Product(str, Enum):
```

Comment:
I prefer this to be nested as you can do check against the Enum without needing to import anything new

Reply:
(str,Enum) makes this a str too so: 

Product("canvas") == Product.CANVAS == "canvas"